context/test_notebooks.py

```diff
@@ -35,6 +37,8 @@
     'cuspatial/ZipCodes_Stops_PiP_cuSpatial.ipynb',
     # context on this being skipped: https://github.com/rapidsai/docker/issues/726
     'cuspatial/trajectory_clustering.ipynb',
+    # context on this being skipped: https://github.com/rapidsai/docker/issues/740
+    'cuspatial/Taxi_Dropoff_Reverse_Geocoding.ipynb',
```

Comment:
proposing this to unblock CI, see #740

@isVoid I'll start a separate offline thread about our strategy for these notebooks.